src/rust/sdk/src/lib.rs

```diff
@@ -1,3 +1,16 @@
+//! # Econia SDK
+//!
+//! The Econia Rust SDK is an SDK to interact with the Econia V4 protocol. You can read more about
+//! the protocol at <https://econia.dev/>.
+//!
+//! The workflow to use this SDK is quite straight forward (only 3 steps):
+//!
+//! 1. Create a [`EconiaClient`]
+//! 2.
+//!     - Either use it to get a [`EconiaViewClient`]
+//!     - Or create a payload using helper functions from the [`entry`] module
```

Comment:
How about something like?

"To use the SDK create an [`EconiaClient`], then either use it to get an [`EconiaViewClient`], or create a payload using helper functions from the [`entry`] module"